src/main/java/org/mule/extension/api/annotation/param/metadata/Content.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.annotation.param.metadata;
+
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import org.mule.extension.api.annotation.metadata.MetadataScope;
+import org.mule.extension.api.metadata.MetadataResolver;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Marks the parameters as the main input of an Operation and as the parameter to provide
```

Comment:
parameters -> parameter.... only one argument can bare this annotation. Make sure to make that clear on the javadoc